src/types.ts

```diff
@@ -82,6 +96,7 @@ export interface PrefabComponent {
       textTransform?: string;
     };
   };
+  descendants: PrefabReference[];
```

Comment:
`structure` on line 177 of this file probably also need to be typed as `PrefabReference[]`?

Reply:
If so, then you probably want to write tests for that use-case as well.

Reply:
we have also implemented the PrefabReference type within the structure of the prefab.